### PR TITLE
[CLI][Stake][e2e][1/n] Add init/add staking e2e

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Testing the following commands

1. initialize-stake-owner
2. add-stake

### Test Plan

```

2023-07-21 16:36:54,698 - INFO - These tests passed:
2023-07-21 16:36:54,698 - INFO - test_metrics_accessible
2023-07-21 16:36:54,698 - INFO - test_init
2023-07-21 16:36:54,698 - INFO - test_config_show_profiles
2023-07-21 16:36:54,698 - INFO - test_account_fund_with_faucet
2023-07-21 16:36:54,698 - INFO - test_account_create
2023-07-21 16:36:54,698 - INFO - test_account_lookup_address
2023-07-21 16:36:54,698 - INFO - test_aptos_header_included
2023-07-21 16:36:54,698 - INFO - test_move_compile
2023-07-21 16:36:54,698 - INFO - test_move_compile_script
2023-07-21 16:36:54,698 - INFO - test_move_publish
2023-07-21 16:36:54,698 - INFO - test_move_run
2023-07-21 16:36:54,698 - INFO - test_move_view
2023-07-21 16:36:54,698 - INFO - test_stake_initialize_stake_owner
2023-07-21 16:36:54,698 - INFO - test_stake_add_stake
2023-07-21 16:36:54,698 - INFO - test_node_show_validator_set
2023-07-21 16:36:54,698 - INFO - test_account_rotate_key
2023-07-21 16:36:54,698 - INFO - All tests passed!
```

### CODE REVIEW

File: crates/aptos/e2e/cases/stake.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "initialize-stake-owner",
+            "--initial-stake-amount",
+            "100000000000",
+            "--assume-yes",
+        ],
+    )
+
+    result = json.loads(response.stdout)["Result"]
+    if result.get("success") != True:
+        raise TestError("Did not initialize stake owner successfully")
```
Comment (gregnazario) [inline crates/aptos/e2e/cases/stake.py:28]: We should probably do more than just check that it's a success, looking for that it has fields that we want in the output (maybe not exactly what they say)

**Review by gregnazario — APPROVED** [review]
It looks good, I just think we can do better in general on the verification at the end to prevent breaking changes.

Though, that also then brings the question of how inflexible we want to be

**Comment by 0xjinn** [comment]
I updated the fix on comments on #9284 Originally I was try to make it more Stack-PR style, but it just make things more complicated. 

Close this PR now

File: crates/aptos/e2e/cases/stake.py (hunk 1 above)
Comment (0xjinn) [reply crates/aptos/e2e/cases/stake.py:28]: Resolved on #9284

**Pull request closed without merging**